### ISSUE TITLE
2 个私库之间是是不能同步的吧

### ISSUE THREAD

**Issue by Leader755** [wearerequired/git-mirror-action#12]
我试了很多次，不知道是配置问题还是，2 个私库之间同步一直是失败的。github私库到 gitee 私库，请教大佬了

**Comment by ocean90** [comment]
Yes, you can use it to sync private repositories. You only have to make sure that the SSH key has access to both.

I'm not familiar with Gitee so you have to find something similar to GitHub's deploy keys. For GitHub this is documented [here](https://docs.github.com/en/developers/overview/managing-deploy-keys#setup-2). For Gitee you also have to make sure that write access is enabled.

**Comment by ocean90** [comment]
Closing due to lack of feedback. Note that I have improved the documentation in https://github.com/wearerequired/git-mirror-action/commit/a3d698669230bf295361234b13e6cdcbc0036e21 a bit.
